lazygit.hcl

```diff
@@ -0,0 +1,50 @@
+description = "simple terminal UI for git commands"
+binaries = ["lazygit"]
+source = "https://github.com/jesseduffield/lazygit/releases/download/v${version}/lazygit_${version}_${os_}_${arch_}.tar.gz"
+
+platform "amd64" {
+  vars = {
+    "arch_": "x86_64",
+  }
+}
+
+platform "linux" {
+  vars = {
+    "os_": "Linux",
+  }
+}
+
+platform "darwin" {
+  vars = {
+    "os_": "Darwin",
```

Comment:
You typically don't need to worry about case changes s as GH is case insensitive

Reply:
ah good to know